scio-grpc/src/main/scala/com/spotify/scio/grpc/SCollectionSyntax.scala

```diff
@@ -92,6 +92,65 @@ class GrpcSCollectionOps[Request](private val self: SCollection[Request]) extend
       .mapValues(_.asScala.map(_.asScala))
   }
 
+  def batchGrpcLookup[
+    BatchRequest: Coder,
+    BatchResponse: Coder,
+    Output: Coder,
+    Client <: AbstractFutureStub[Client]
+  ](
+    channelSupplier: () => Channel,
+    clientFactory: Channel => Client,
+    batchSize: Int,
+    batchRequestFn: Seq[Request] => BatchRequest,
+    batchResponseFn: BatchResponse => Seq[(String, Output)],
+    idExtractorFn: Request => String,
+    maxPendingRequests: Int,
+    cacheSupplier: CacheSupplier[String, Output] = new NoOpCacheSupplier[String, Output]()
+  )(
+    lookupFn: Client => BatchRequest => ListenableFuture[BatchResponse]
+  )(implicit
+    requestCoder: Coder[Request],
```

Comment:
this coder should be available from the `self` SCollection

Reply:
Ohh .. didn't know that. 
I unfortunately have a very shallow understanding of Coders. I understand what they do (help serialize types) but i don't understand how they propagate. Had the same the issue in Flink when i used it before as well :( 
if you have a good resource would be awesome (i'll definitely look up as well as it's time to get educated about the topic)

anyway .. i added `import self.coder` which fixed the issue.